source/common/http/user_agent.cc

```diff
@@ -21,7 +21,8 @@ void UserAgent::completeConnectionLength(Stats::Timespan& span) {
     return;
   }
 
-  scope_->histogram(prefix_ + "downstream_cx_length_ms").recordValue(span.getRawDuration().count());
+  scope_->histogram(prefix_ + "downstream_cx_length", Stats::Histogram::Unit::Milliseconds)
```

Comment:
can you do me a favor and put:

// TODO(jmarantz): use stat-names here.

Reply:
Done.